scripts/generateRedirects.js

```diff
@@ -0,0 +1,34 @@
+const { resolve } = require('path');
+const { readFileSync, writeFile } = require('fs');
+const redirects = require('../util-node/redirects');
+
+const firebaseConfig = resolve(__dirname, '../firebase.json');
+
+const firebaseJSON = JSON.parse(readFileSync(firebaseConfig));
+
+let firebaseRedirects = [];
+
+firebaseRedirects = Object.entries(redirects).map(([key, value]) => ({
+  ...firebaseRedirects,
+  source: key,
+  destination: value,
+  type: '301',
+}));
```

Comment:
This spread seems unnecessary. We can do a const here I think.

```suggestion
const firebaseRedirects = Object.entries(redirects).map(([key, value]) => ({
  source: key,
  destination: value,
  type: '301',
}));
```

Reply:
Agreed